src/ts/component/menu/smile.tsx

```diff
@@ -86,7 +91,7 @@ class MenuSmile extends React.Component<I.Menu, State> {
 				const Item = (item: any) => {
 					const str = `:${item.itemId}::skin-tone-${item.skin}:`;
 					return (
-						<div 
+						item.itemId === ID_BLANK ? <div className="item"></div> : <div 
```

Comment:
both are strings, no need for strict equality, ==, use self-closing div, all empty elements should be self-closing